feeds/wlan-ap/opensync/src/platform/openwrt/src/lib/target/src/stats.c

```diff
@@ -275,6 +275,21 @@ bool target_stats_survey_convert(radio_entry_t *radio_cfg, radio_scan_type_t sca
 {
 	target_survey_record_t delta;
 
+	if (data_new->info.chan != data_old->info.chan) {
+		/* Do not consider the old channel's data */
+		memset(data_old, 0, sizeof(*data_old));
+	} else if ((data_new->duration_ms <= data_old->duration_ms) &&
+			(data_new->chan_busy <= data_old->chan_busy)) {
```

Comment:
Do we need to worry about a false trigger of this code due to a counter wrap?

Reply:
It is highly unlikely to have both the counters (duration_ms and chan_busy) wrap at the same time.